inc/formanswer.class.php

```diff
@@ -938,6 +940,10 @@ public function generateTarget() {
                $success = false;
                break;
             }
+            $this->targetList[] = [
+               'itemtype' => $generatedTarget->getType(),
+               'items_id' => $generatedTarget->getID()
+            ];
```

Comment:
Why not add the object instead of an array itemtype / items_id ? This way you could avoid the use of call_user_func_array in front/form.form.php

```suggestion
            $this->targetList[] = $generatedTarget;
```

Reply:
seems a good idea

Reply:
Fixed ;)